bin/binaryaudit

```diff
@@ -92,7 +94,7 @@ elif "poky" == args.cmd:
 
     if args.compare_buildhistory:
         suppr = []
-        if args.global_suppression is not None:
+        if args.global_suppression is not None and use_suppressions:
```

Comment:
Not sure this is the intended behavior. As it were thought the behavior to be common, the poky part should have same behavior as mariner does - it would include default suppressions by default until it's prohibited by the option. Otherwise passing suppressions and disabling them at the same time doesn't seem logic.

At the same time, Mariner would make use of a possibility to pass an additional suppressions file. So this would round up the functionality and make it equivalent for both. 

Thanks